src/components/DomainBadge.tsx

```diff
@@ -12,7 +12,7 @@ export type DomainBadgeProps = {
   projectId?: number
 }
 
-function getJuicecrowdUrl(projectId: number) {
+export function getJuicecrowdUrl(projectId: number) {
```

Comment:
This should probably be moved somewhere better where it makes sense

Reply:
Good call sir, have moved to a `utils/juicecrowd.ts` file